src/Tribe/Views/V2/Customizer/Section/Global_Elements.php

```diff
@@ -0,0 +1,510 @@
+				'label'       => esc_html__( 'Background Color', 'the-events-calendar' ),
+				'description' => esc_html__( 'All calendar and event pages', 'the-events-calendar' ),
+				'choices'     => [
+					'transparent' => _x(
+						'Transparent.',
+						'Label for option to leave transparent (default).',
+						'the-events-calendar'
+					),
+					'custom'	  => esc_html_x(
+						'Select Custom Color',
+						'Label for option to set a custom color.',
+						'the-events-calendar'
+					),
+				],
+			],
+			'background_color'        => [
+				'priority'  => 26, // Should come right after background_color_choice
+				'type'      => 'color',
+			],
+			'accent_color'            => [
+				'priority' => 30,
+				'type'     => 'color',
+				'label'    => esc_html_x(
+					'Accent Color',
+					'The event accent color setting label.',
+					'the-events-calendar'
+				),
+			],
+		];
+	}
+
+	/**
+	 * Filters the Global Elements section CSS template to add Views v2 related style templates to it.
+	 *
+	 * Please note: the order is important for proper cascading overrides!
+	 *
+	 * @since 5.3.1
+	 *
+	 * @param string                      $css_template The current CSS template, as produced by the Section.
+	 * @param \Tribe__Customizer__Section $section      The Global Elements section.
+	 * @param \Tribe__Customizer          $customizer   The current Customizer instance.
+	 *
+	 * @return string The filtered CSS template.
+	 */
+	public function get_css_template( $css_template ) {
+		// For sanity's sake.
+		if ( ! tribe_events_views_v2_is_enabled() ) {
+			return $css_template;
+		}
+
+		$new_styles   = [];
+
+		// It's all custom props now, baby...
+
+		// Font family override.
+		if ( $this->should_include_setting_css( 'font_family' ) ) {
+			$new_styles[] = "--tec-font-family-sans-serif: inherit;";
+			$new_styles[] = "--tec-font-family-base: inherit;";
+		}
+
+		/**
+		 * It's about to get complicated - Font Size overrides!
+		 *
+		 * If they set a base font size, we set the font_size slider to match via js.
+		 * If they set the slider, we set `font_size_base` to match via js.
+		 * So we only have to do calculations based on the `font_size_base` setting.
+		 *
+		 * Original font sizes for reference:
+		 * --tec-font-size-0: 11px;
+		 * --tec-font-size-1: 12px;
+		 * --tec-font-size-2: 14px;
+		 * --tec-font-size-3: 16px;
+		 * --tec-font-size-4: 18px;
+		 * --tec-font-size-5: 20px;
+		 * --tec-font-size-6: 22px;
+		 * --tec-font-size-7: 24px;
+		 * --tec-font-size-8: 28px;
+		 * --tec-font-size-9: 32px;
+		 * --tec-font-size-10: 42px;
+		 */
+		if ( $this->should_include_setting_css( 'font_size_base' ) ) {
+			$sizes           = [ 11, 12, 14, 16, 18, 20, 22, 24, 28, 32, 42, ];
+			$size_multiplier = 1;
+			$size_multiplier = round( (int) $this->get_option( 'font_size_base' ) / 16, 3 );
+
+			foreach ( $sizes as $key => $size ) {
+				$font_size = $size_multiplier * (int) $size;
+				// round to whole pixels.
+				$font_size = round( $font_size );
```

Comment:
just asking the question, but is there a need for this to be a whole number? can it be a non-whole number / rounded to a decimal?

Reply:
It could be - but most folks think of font sizes in whole pixels?
Also I'm expecting this to change in the future when we convert to em/rem?